common/Generieke-Datatypen-Gemeenten.yaml

```diff
@@ -0,0 +1,79 @@
+openapi: 3.0.0
```

Comment:
Wat is de ratio achter het onderscheid tussen de common.yaml (van HC) en dit bestand? Waarom horen de componenten niet gewoon bij elkaar in 1 schema?

Reply:
Dit bestand wordt echt een 1 op 1 vertaling van het Generieke Datatypen Gemeenten basismodel naar een JSON-schema. in de common.yaml nemen we ook zaken als herbruikbare parameters en foutmeldingen etc op.

Reply:
Moeten we in 'Generieke Datatypen Gemeenten' of in een ander model niet ook nog componenten opnemen voor:
* NietNegatiefInteger
* POSTCODE
* YearMonth

En dekt het datatype 'DatumOnvolledig' de datatypes 'DMO' en 'DTMO' helemaal af?

Reply:
Dat kan, maar dat is een ander issue dan in dit pull request wordt geregeld. Gaarne een issue inschieten.....

Reply:
Doe ik.